src/web/core/src/main/java/org/geoserver/web/data/layergroup/LayerGroupProvider.java

```diff
@@ -31,17 +57,11 @@ public class LayerGroupProvider extends GeoServerDataProvider<LayerGroupInfo> {
     static final Property<LayerGroupInfo> CREATED_TIMESTAMP =
             new BeanProperty<>("datecreated", "dateCreated");
 
-    public static Property<LayerGroupInfo> ENABLED =
-            new AbstractProperty<LayerGroupInfo>("Enabled") {
-
-                @Override
-                public Boolean getPropertyValue(LayerGroupInfo item) {
-                    return Boolean.valueOf(item.isEnabled());
-                }
-            };
+    public static Property<LayerGroupInfo> ENABLED = new BeanProperty<>("enabled", "enabled");
 
     static List<Property<LayerGroupInfo>> PROPERTIES = Arrays.asList(NAME, WORKSPACE, ENABLED);
 
+    /** revisit: this seems to be dead code, there're no uses of this interface in the code base */
     protected LayerGroupProviderFilter groupFilter = null;
```

Comment:
This can indeed be removed, it was used by the WMS-EO extension, which was eventually demoted down to community module, and then removed altogether. It can be safely removed, and then you can simplify the code a bit.

Reply:
done